packages/theme/package.json

```diff
@@ -15,5 +15,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
```

Comment:
Should this be a devDependency instead?

Reply:
Nah, chalk` is used by the template at `enact create` time, it's needed for operation of the template.